### PR TITLE
Rebuild for 2021.8.26 

### PR DESCRIPTION
**Pull request by ryanskeith**
Bumped versions and changed tests.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ build:
     - libwebp  # [win or (osx and arm64)]
```
Comment (cbouss) [inline recipe/meta.yaml:22]: build number should become 
number: 1

**Review by cbouss — COMMENTED** [review]
If the build number is updated, looks good to me.

File: recipe/meta.yaml (hunk 1 above)
Comment (ryanskeith) [reply recipe/meta.yaml:22]: Another victim of the merge mess I created.  Just bumped it.

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**